### DISCUSSION TITLE
Avoid em dash conversion in `fmt_markdown`

### DISCUSSION THREAD

**Discussion by wvictor14** [rstudio/gt#1613] [Q&A] [answered]
Is there a way to avoid em dash conversion in fmt_markdown?

`tibble(x = '---') |> gt() |>  fmt_markdown() `

![image](https://github.com/rstudio/gt/assets/22013187/f640a767-643b-4c3a-afaf-ff805f0e7dcb)

reason I want to use fmt_markdown is for newlines to be introduced via `<br>`

**Comment by olivroy** [comment, marked as answer]
You'd have to do a manual transformation of your input before.


```r
# this does what you want.
dplyr::tibble(x = '\\-\\-\\-') |> gt() |>  fmt_markdown() 
```

Programatically, you can do 

```r

dplyr::tibble(x = '---') |> dplyr::mutate(x = gsub("-", "\\\\-", x))|>  gt() |>  fmt_markdown() 
# or with stringr
dplyr::tibble(x = '---') |> dplyr::mutate(x = stringr::str_replace_all(x, "-", "\\\\-"))|>  gt() |>  fmt_markdown() 
```
![image](https://github.com/rstudio/gt/assets/52606734/42bb61b3-24b6-4173-a85c-e680bf28069d)

The problem is that `---` is em-dash in markdown, so you need to make sure it looks like `\-` to avoid markdown interpretation.
